change first page

nick1udwig opened the pull request:
- [x] Simplify intro pages
- [x] Simplify installation

nick1udwig commented:
Close on `Simplify installation`: thinking about using https://github.com/kinode-dao/kit/pull/174 but need to confirm its behavior on MacOS

dr-frmr commented:
Changes here look good so far

nick1udwig commented:
PTAL @dr-frmr

dr-frmr commented on src/login.md:
```diff
@@ -25,9 +24,11 @@ Arguments:
 Options:
       --port <PORT>            Port to bind [default: first unbound at or above 8080]
       --ws-port <PORT>         Kinode internal WebSockets protocol port [default: first unbound at or above 9000]
+      --tcp-port <PORT>        Kinode internal TCP protocol port [default: first unbound at or above 9000]
```
should be 10000 now, changed this for hosting platform purposes

dr-frmr commented on src/login.md:
```diff
@@ -25,9 +24,11 @@ Arguments:
       --verbosity <VERBOSITY>  Verbosity level: higher is more verbose [default: 0]
       --reveal-ip              If set to false, as an indirect node, always use routers to connect to other nodes.
       --rpc <RPC>              Add a WebSockets RPC URL at boot
+      --password <PASSWORD>    Node password
```
i would put a note here to use single quotes '' around the password, because i was scratching my head for like 20 minutes when i didn't do this and special chars in my password got mangled

nick1udwig replied:
That should be added to help text if it hasn't been already

dr-frmr reviewed (commented):
2 small changes then merge away

nick1udwig commented:
Addressed your comments (& updated helptext of Kinode core https://github.com/kinode-dao/kinode/pull/397/commits/b907b7d0e69c8387c298b34dbb7e1d0e59a35d97)
